source/manifest.json

```diff
@@ -1,4 +1,10 @@
 {
+  "applications": {
+    "gecko": {
+      "id": "theoldreader@knyar",
+      "strict_min_version": "53.0"
+    }
+  },
```

Comment:
This is where we hit my knowledge. Never worked with npm or grunt. But https://www.npmjs.com/package/grunt-webext-builder and https://www.npmjs.com/package/grunt-crx look promising. @kav2k if you have experience with grunt, can you try those?